modules/tinymce/src/themes/silver/main/ts/ui/core/ChoiceControls.ts

```diff
@@ -104,7 +109,7 @@ const lineHeightSpec: ControlSpec<string> = {
   display: Fun.identity,
 
   getCurrent: (editor) => Optional.from(editor.queryCommandValue('LineHeight')),
-  setCurrent: (editor, value) => editor.execCommand('LineHeight', false, value)
+  setCurrent: (editor, value) => editor.execCommand('LineHeight', false, value),
```

Comment:
Is that a trailingComma?!

Reply:
Oops, left that in... in saying that though we're fine with trailing commas :shrug: 
```suggestion
  setCurrent: (editor, value) => editor.execCommand('LineHeight', false, value)
```